.travis.sh

```diff
@@ -1,19 +1,14 @@
 #!/usr/bin/env bash
 
-DOCKER_IMAGE_CLIENT=uumpa/hasadna-openpension:client
-DOCKER_IMAGE_SERVER=uumpa/hasadna-openpension:server
-
 if [ "${1}" == "deploy" ]; then
-    docker tag hasadna/open-pension-server ${DOCKER_IMAGE_SERVER}-latest &&\
-    docker tag hasadna/open-pension-server ${DOCKER_IMAGE_SERVER}-${TRAVIS_COMMIT} &&\
-    docker tag hasadna/open-pension-client ${DOCKER_IMAGE_CLIENT}-latest &&\
-    docker tag hasadna/open-pension-client ${DOCKER_IMAGE_CLIENT}-${TRAVIS_COMMIT} &&\
-    docker push ${DOCKER_IMAGE_CLIENT}-latest &&\
-    docker push ${DOCKER_IMAGE_CLIENT}-${TRAVIS_COMMIT} &&\
-    docker push ${DOCKER_IMAGE_SERVER}-latest &&\
-    docker push ${DOCKER_IMAGE_SERVER}-${TRAVIS_COMMIT} &&\
+    cd /home/travis/build/hasadna/open_pension || true && \
```

Comment:
Let's call the file `deploy.sh` and in travis we need to do something like https://github.com/hasadna/open-pension-static-site/blob/master/.travis.yml#L10-L14

Reply:
I think it's safer this way